vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -356,6 +357,18 @@ $requirements = @(
         Install = {
             & choco install -y dotnetcore-3.1-sdk
         }
+    },
+    @{
+        ID=[CheckId]::NuGet
+        Name = 'NuGet'
+        Tags = @('appDev')
+        Valid = {
+            Get-Command -Name NuGet.exe -ErrorAction SilentlyContinue -OutVariable cmd
+            $cmd -and '5.8.0.0' -le $cmd.FileVersionInfo.FileVersion
+        }
+        Install = {
+            & choco install -y NuGet.CommandLine --version 5.8.0
```

Comment:
This should probably be a yarn/npm command installing local package `nuget-exe`.

Reply:
We aren't doing any user installation here. The exe is just downloaded as part of yarn install. There is no user facing action to do, or warn about.

Reply:
Because Yarn takes care of this, I'll just remove the section altogether.